common/models/user.js

```diff
@@ -301,7 +301,9 @@ module.exports = function(User) {
    * Compare the given `password` with the users hashed password.
    *
    * @param {String} password The plain text password
-   * @returns {Boolean}
+   * @callback {Function} callback Callback function
+   * @param {Error} err Error object
+   * @param {Boolean} isMatch Returns true if the given `password` matches record
```

Comment:
thank you @superkhau and yeah I have the same question when I edit the docs, I am confused with the naming, like in function [`login`](https://github.com/strongloop/loopback/blob/doc/user-hasPassword/common/models/user.js#L169) it also uses `callback` but the the parameter in `login` called `fn`
And same here in [`logout`](https://github.com/strongloop/loopback/blob/doc/user-hasPassword/common/models/user.js#L282)

And I use `password` as it original is.
I guess the doc names are easy to understand by users.